tokio-threadpool/src/pool/backup.rs

```diff
@@ -58,7 +58,7 @@ pub(crate) enum Handoff {
 }
 
 /// Tracks thread state.
-#[derive(Clone, Copy, Eq, PartialEq)]
+#[derive(Clone, Copy, Eq, PartialEq, Default)]
 struct State(usize);
```

Comment:
Hmm... `clippy::new_without_default` seems to warn the private type by mistake.

Reply:
Related: https://github.com/rust-lang/rust-clippy/issues/1878#issuecomment-314078003

Reply:
Fixed in https://github.com/tokio-rs/tokio/pull/1335/commits/063c4d255f48876ac9fb4e608806fecb179d201e.